_posts/2025-05-25-update.md

```diff
@@ -7,6 +7,12 @@ slug: 2025-05-25-update
 
 ## Developer News
 
+The [Program Committee is now accepting applications](https://docs.google.com/forms/d/e/1FAIpQLScUIRKNaqb1fuNZB4vpfw05fA2wvsHqUWUd9t6nY718pf4WiQ/viewform) for the Maintainer Summit North America 2025! Share your interest in joining the committee before Monday, July 7th.
+
+Kubespray v2.28.0 has been released. Please refer to the [release page](https://github.com/kubernetes-sigs/kubespray/releases/tag/v2.25.0) for more information. 
+
+The Release Cycle for 1.34 has started, and the release team is actively [collecting enhancements](https://github.com/orgs/kubernetes/projects/213/views/1). SIG Leads are to discuss enhancements to be proposed for this release and add the "lead-opted-in" label in the [Enhancements repository](https://github.com/kubernetes/enhancements/issues).
```

Comment:
@siddhant-khisty The kubespray item would go in the subproject updates and release cycle update would be better suited for release schedule section rather than dev news. I'll merge and fix these now, but noting for future reference.